charts/timed/values.yaml

```diff
@@ -72,6 +72,50 @@ backend:
     serverEmail: webmaster@chart-example.local
     # backend.settings.workReportPath -- Path where the workreport shall be loaded from. The contents of the default path is filled from `configmap-workreport.yaml`.
     workReportPath: /etc/workreport
+  cronjobs:
+    # backend.cronjobs.notifySupervisors -- Notify supervisors
+    notifySupervisors:
+      schedule: "0 8 * * 4"
+      command:
+        - "./manage.py"
+        - "notify_supervisors_shorttime"
+    # backend.cronjobs.notifyReviewersFirst -- Notify reviewers first stage
+    notifyReviewersFirst:
+      schedule: "0 8 4 * *"
+      command:
+        - "./manage.py"
+        - "notify_reviewers_unverified"
+        - "--offset 5"
+    # backend.cronjobs.notifyReviewersSecond -- Notify reviewers second stage
+    notifyReviewersSecond:
+      schedule: "0 8 11 * *"
+      command:
+        - "./manage.py"
+        - "notify_reviewers_unverified"
+        - "--offset 5"
+        - "--message"
+        - "'Bitte verbleibende Positionen verifizieren.'"
```

Comment:
Maybe this should have some docs since it's reasonable to expect that it will be changed in non de cases.